modules/ROOT/pages/websocket/websocket-connector-config-topics.adoc

```diff
@@ -0,0 +1,285 @@
+= WebSocket Additional Configuration Information - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+== Configure WebSocket Server
+
+If your app exposes a WebSocket inbound endpoint, you need to set that up using thhe `<ws:server-settings>` element.
+
+Because WebSocket connections are initiated with an HTTP Request, exposing a WebSockets inbound endpoint requires pointing to an underlying HTTP listener configuration. This not only allows you to reuse all the available features and configurations of an HTTP listener (such as host, port, TLS, etc.), it also allow your app to expose both WebSockets and regular HTTP endpoints using the same port.
```

Comment:
```suggestion
Because WebSocket connections are initiated with an HTTP Request, exposing a WebSockets inbound endpoint requires pointing to an underlying HTTP listener configuration. This not only allows you to reuse all the available features and configurations of an HTTP listener (such as host, port, TLS, etc.), it also allows your app to expose both WebSockets and regular HTTP endpoints using the same port.
```